packages/components/input/src/input.vue

```diff
@@ -443,7 +443,37 @@ watch(
   }
 )
 
+// Get the widths of 'suffix' and 'prefix' to set the padding property of the input
+// https://github.com/element-plus/element-plus/issues/6464
+const innerSuffixRef = ref<HTMLElement>()
+const innerPrefixRef = ref<HTMLElement>()
+const inputStyleInner = ref<CSSProperties>({})
+const getSuffixOrPrefixWidth = (
+  slotElm: Ref<HTMLElement>,
+  defaultVal: number
+): number => {
+  if (slotElm.value) {
+    const slotElmWidth = (slotElm.value as HTMLElement).offsetWidth
+    return slotElmWidth > 0 ? slotElmWidth + 16 : defaultVal
+  }
+  return defaultVal
+}
+const setInputPadding = (): void => {
+  nextTick(() => {
+    // 如果用户传入了 padding 就用用户的覆盖
```

Comment:
Can you change this to an English comment?

Reply:
I have modified this comment in English

Reply:
> Can you change this to an English comment?

I have modified this comment in English